.github/workflows/ci.yml

```diff
@@ -0,0 +1,93 @@
+name: Test
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: |
+          yarn global add lerna
+          # approach taken from https://github.com/actions/setup-node/issues/85
+          echo "$(yarn global bin)" >> $GITHUB_PATH
+      - run: lerna bootstrap
+      - run: lerna run test
```

Comment:
I'm not sure if it's a good idea to run tests for all packages to deploy one of them, to run tests for the interface only you can do `lerna run test --scope @gnosis.pm/safe-app-developer-ui`

Reply:
Yes, I would just run tests specific to the packages we're deploying

Reply:
Just changed Test job on the CI to run tests for safe-app-developer-ui

Reply:
@giacomolicari but the script that previously tested all packages is still removed. Could you please restore it?